device/accton/x86_64-accton_as9726_32d-r0/pmon_daemon_control.json

```diff
@@ -1,5 +1,5 @@
 {
     "skip_ledd": true,
-    "skip_thermalctld": true
+    "skip_pcied": true
```

Comment:
Is there any specific reason to skip the pcied?

Reply:
There is no pcie devices, so we skip.